code/modules/assembly/pass.dm

```diff
@@ -0,0 +1,40 @@
+var/list/PASS_list = list()
```

Comment:
does this list actually ever get used? I don't see it being added to or removed from

Reply:
It was used originally when I had them have built in GPS. Lemme remove it.